sphinx_book_theme/__init__.py

```diff
@@ -125,10 +126,18 @@ def generate_nav_html(
             master_doc = config["master_doc"]
             master_doctree = app.env.get_doctree(master_doc)
             master_url = context["pathto"](master_doc)
-            master_title = list(master_doctree.traverse(nodes.title))
-            if len(master_title) == 0:
-                raise ValueError(f"Landing page missing a title: {master_doc}")
-            master_title = master_title[0].astext()
+
+            # check title in globaltoc (jb case) else get title from doctree
+            master_title = None
+            with suppress(Exception):
+                master_title = app.config["globaltoc"].get("title")
```

Comment:
note this won't work quite the same with https://github.com/executablebooks/jupyter-book/pull/1293. But I would be interested if there is a less theme specific solution that could go straight into sphinx-external-toc 🤔

Reply:
@chrisjsewell you mean creating a global variable in sphinx-external-toc which has this value? and can be accessed directly in any theme.